Cargo.toml

```diff
@@ -199,6 +200,10 @@ filesystem_watcher = ["bevy_internal/filesystem_watcher"]
 # Enable serialization support through serde
 serialize = ["bevy_internal/serialize"]
 
+# Enables multithreaded parallelism in the engine. Disabling it forces all engine 
+# tasks to run on a single thread.
+multi-threaded = ["bevy_internal/multi-threaded"]
```

Comment:
I guess it's still failing because multi line is not handled correctly, could you try putting it all in one line?

Reply:
Ah, good catch, wasn't understanding why it was adding the `#` there.